app/konnect/dev-portal/access-and-approval/dev-reg.md

```diff
@@ -20,21 +20,39 @@ All Developers must register through the {{site.konnect_short_name}} [Dev Portal
 
 2. At the bottom of the login form, click **Sign Up**.
 
-3. Fill out the registration form and click **Create Account**. 
+3. Fill out the registration form and click **Create Account**.
 
    You should receive an email to confirm your email address. Click the verification link in the email.
 
-5. Set a password for your account.
+4. Set a password for your account.
 
-If your admin has auto-approve enabled, you can log into the Dev Portal immediately after setting a password. Learn how to [set up auto-approval for developer access as an admin](/konnect/dev-portal/access-and-approval/auto-approve-devs-apps/).
+### SSO Login
+
+1. Navigate to the {{site.konnect_short_name}} Dev Portal URL provided by your {{site.konnect_short_name}} admin.
+
+2. At the top of the login form, click **Login with SSO**.
 
-If auto-approve is not enabled, an admin has to review and approve your request manually. All manually approved developers are notified via email upon gaining access to the Dev Portal. Learn how to [manually approve developer access as an admin](/konnect/dev-portal/access-and-approval/manage-devs/#approve-dev-reg).
+3. You will be redirected to the Identity Provider (IdP) that the admin has configured (e.g. [Okta SSO](https://www.okta.com/products/single-sign-on/)). Here, you will sign in using your IdP's credentials.
+
+4. Once you've logged in, you will be redirected to the {{site.konnect_short_name}} catalog.
 
 ## Account Management {#management}
 
 If you attempt to register multiple times with the same email address, you’ll receive an email with the status of your registration. [Possible statuses](/konnect/dev-portal/access-and-approval/manage-devs/#developer-status) are: `Pending`, `Rejected`, `Revoked`, and `Approved`.
 
 If you forget your password, click on "Forgot your password?" and enter the email address associated with your Dev Portal account. If an account exists with the email address provided, you’ll receive an email prompting you to change your password.
 
-Contact your {{site.konnect_short_name}} admin if you have login issues.
+If your admin has auto-approve enabled, you can log into the Dev Portal immediately after setting a password. Learn how to [set up auto-approval for developer access as an admin](/konnect/dev-portal/access-and-approval/auto-approve-devs-apps/).
 
+### Basic Auth Login
```

Comment:
I just realized - isn't it duplication of `## Register as a Developer {#register}` section above? Do we need it at all?